pytest_mh/utils/fs.py

```diff
@@ -207,6 +207,39 @@ def write(
             log_level=SSHLog.Error,
         )
 
+    def append(
+        self,
+        path: str,
+        contents: str,
+        *,
+        dedent: bool = True,
+    ) -> None:
+        """
+        Append to a remote file.
+
+        :param path: File path.
+        :type path: str
+        :param contents: File contents to write.
+        :type contents: str
+        :param dedent: Automatically dedent and strip file contents, defaults to True
+        :type dedent: bool, optional
+        """
+        if dedent:
+            contents = textwrap.dedent(contents).strip()
+
+        self.backup(path)
+        self.logger.info(f'Writing file "{path}" on {self.host.hostname}', extra={"data": {"Contents": contents}})
+
+        self.host.ssh.run(
+            f"""
+                set -ex
+                cat >> '{path}'
+                {self.__gen_chattrs(path)}
```

Comment:
Since we do not set mode, user and  group we do not need this line.

Reply:
Fixed